gradle/build-hadoop-distros.gradle

```diff
@@ -28,6 +28,7 @@ project('spring-xd-hadoop:hadoop24') {
 		runtime ("org.apache.hadoop:hadoop-mapreduce-client-jobclient:$hadoop24Version")
 		runtime ("org.apache.hadoop:hadoop-streaming:$hadoop24Version")
 		runtime ("org.apache.hadoop:hadoop-yarn-common:$hadoop24Version")
+		configurations.runtime.exclude(group: 'org.xerial.snappy')
```

Comment:
The only reason we need these excludes are for our tests to run in an IDE, so for the pom files we really don't need theese excludes